app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,6 @@
+class GuestsController < ApplicationController
+  def show
+    @guest = Guest.find(params[:id])
+    @rooms = @guest.rooms
+  end
```

Comment:
We could remove line 4 and just pass `@guest` to the view since the guest object knows about the rooms associated with it. We want to send as few variables to the view as we can, so always consider "Is this information available with any other objects I'm sending off to the view?"